Sources/Testing/EntryPoints/ABIEntryPoint.swift

```diff
@@ -28,7 +28,7 @@
 ///
 /// - Warning: This function's signature and the structure of its JSON inputs
 ///   and outputs have not been finalized yet.
-@_spi(ForToolsIntegrationOnly)
+@_spi(Experimental) @_spi(ForToolsIntegrationOnly)
```

Comment:
Why `@_spi(Experimental)`

Reply:
Because we haven't gone through an API review process yet, nor have we pinned down the interface for good.